src/patternfly/components/Select/select.scss

```diff
@@ -28,13 +28,22 @@
   --pf-c-select__toggle--m-plain--Color: var(--pf-global--Color--200);
   --pf-c-select__toggle--m-plain--hover--Color: var(--pf-global--Color--100);
 
+  // Toggle wrapper doesn't need top padding unless there's a typeahead with adjustment for the input
+  --pf-c-select__toggle-wrapper--PaddingTop: 0;
+
+  // When there is a typeahead (input box) adjust the top padding of the wrapper in case it wraps to the second line
+  --pf-c-select__toggle-wrapper--typeahead--PaddingTop: var(--pf-global--spacer--xs);
+
   // Space for the children of the toggle
   --pf-c-select__toggle-wrapper--not-last-child--MarginRight: var(--pf-global--spacer--xs);
 
   // Max-width is needed for ellipsing the text. This leaves enough space for the toggle caret.
   // However, for cases where the clear icon button or a badge are added, the correct width will need to be calculated in script
   --pf-c-select__toggle-wrapper--MaxWidth: calc(100% - var(--pf-global--spacer--lg));
 
+  // Chip group within a toggle-wrapper needs a margin-bottom in case it wraps
+  --pf-c-select__toggle-wrapper--c-chip-group--MarginBottom: 0;
```

Comment:
do you need this? doesn't look like it's doing anything.

Reply:
The chip group comes with a margin of -4px. It's hard to see but you can see the difference when the typeahead input is wrapped below the chip group and you click on the input to get the focus ring. Without this adjustment, the input is exactly at the bottom of the chip group, with it, there's a little space. @kybaker do you want to say which is correct?

Reply:
Ah, gotcha!